### DISCUSSION TITLE
How to link to a tag page?

### DISCUSSION THREAD

**Discussion by fiatjaf** [srid/emanote#415] [Q&A] [answered]
I've seen the syntax for embedding tagged items, but I can't find a way to create a link to a tags page in the body of a note.

**Comment by srid** [comment, marked as answer]
Doesn't `#foo` already work? For example, see `#emanote/syntax/demo` here which is linking to the corresponding tag page: https://emanote.srid.ca/guide/markdown#hash-tags

Or do you mean linking to the tag index? If so, `[Tags](-/tags)` should work, no?

**Comment by fiatjaf** [comment]
Oh, yes, that should work. I just didn't think of trying that and couldn't find any example.

Thanks!

**Reply by srid** [reply]
Okay, let me know what you end up using Emanote for (if public)!
